### PR TITLE
Add whitepapers to resources

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Add whitepapers to resources

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/resources](http://0.0.0.0:8001/resources)
- Filter by whitepapers
- Check it all works

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/2488

### CODE REVIEW

**Comment by Lukewh** [comment]
We should really show some form of message when there are no results:
http://www.ubuntu.com-pr-2522.run.demo.haus/resources?topic=internet-of-things&content=whitepapers

File: webapp/views.py [hunk 1]
```diff
@@ -198,13 +199,14 @@ def _get_resources(self):
...
                 posts = get_json_feed_content(api_url, limit=3)
-                posts_length = len(posts)
-                if 'posts' not in feed_items.keys():
-                    feed_items['posts'] = {}
-
-                feed_items['posts'][group_name] = {}
-                feed_items['posts'][group_name]['posts'] = posts
-                feed_items['posts'][group_name]['group_name'] = group['name']
+                if posts:
+                    posts_length = len(posts)
```
Comment (Lukewh) [inline webapp/views.py]: This doesn't seem to be used so can be removed.

Comment (anthonydillon) [reply webapp/views.py]: Its passed to the `_generate_pagination_queries` function below.

**Comment by anthonydillon** [comment]
@Lukewh Cheers, fixed the no results notification.

**Comment by Lukewh** [comment]
@anthonydillon has there been a regression in vanilla? Notifications seems to be non-full-width? I noticed this when doing some dashboard work too:

screen shot 2017-12-13 at 11 45 33

File: webapp/views.py (hunk 1 above)
Comment (Lukewh) [reply webapp/views.py]: Line 100 checks the length - what happens if posts_length is not defined? You should maybe define it at the top of the function? Not sure how python handles things like that.

Comment (anthonydillon) [reply webapp/views.py]: It creates the var if not defined. I will set it at the top though.

**Comment by anthonydillon** [comment]
@Lukewh The notification issue is a Vanilla issue. Not sure what changed.

**Pull request merged**